node/middlewares/schema.ts

```diff
@@ -34,13 +37,18 @@ const executor = (app: string): AsyncExecutor => async ({
       headers: {
         'Content-Type': 'application/json',
         Authorization: authToken,
+        cookie: headers.cookie,
         'x-vtex-locale': 'en-US',
         'x-vtex-tenant': 'en-US',
       },
       body: JSON.stringify({ query, variables }),
     }
   )
 
+  if (shouldForwardCookie) {
```

Comment:
if this is always true, why adding this if ?

Reply:
it's not always true, for the introspection request we don't want to forward the cookies (here https://github.com/vtex-apps/graphql-gateway/blob/fix/cookies/node/middlewares/schema.ts#L61)

Reply:
Cool, I haven't seen that one. Anyways, the introspection is done only once, so it shouldn't be a problem